Library/Formula/vim.rb

```diff
@@ -74,11 +74,6 @@ def install
       opts << "--enable-luainterp"
     end
 
-    # XXX: Please do not submit a pull request that hardcodes the path
-    # to ruby: vim can be compiled against 1.8.x or 1.9.3-p385 and up.
-    # If you have problems with vim because of ruby, ensure a compatible
-    # version is first in your PATH when building vim.
```

Comment:
> "not an ordinary ruby dep, but one that involves some conditional kung-fu depending on what my system is like, so maybe I can get HB to look at my special ruby location if I bug them".

Well, this is basically what's going on when you build from source. There's just no need to bug us about it at this point, as we'll use whatever happens to be first in the $PATH. Nonetheless, I've added a small explanatory comment further down.